addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -104,7 +104,19 @@ const Wysiwyg = Widget.extend({
      * @override
      */
     start: async function () {
-        const _super = this._super;
+        await this._super.apply(this, arguments);
+        // If this widget is started from the OWL legacy component, at the time
+        // of start, the $el is not in the document yet. Some instruction in the
+        // start rely on the $el being in the document at that time, including
+        // code for the collaboration (for adding cursors) or the iframe loading
+        // in mass_mailing.
+        if (this.options.startLater) {
```

Comment:
Can't `wysiwyg` detect in which context it is instantiated and react accordingly ?